dotnetv3/Aurora/README.md

```diff
@@ -1,88 +1,131 @@
-* [Describe options for DB instances](Actions/AuroraWrapper.cs)(`DescribeOrderableDBInstanceOptionsAsync`)
-* [Get parameters from a DB cluster parameter group](Actions/AuroraWrapper.cs)(`GetDBClusterParametersAsync`)
-* [Update parameters in a DB cluster parameter group](Actions/AuroraWrapper.cs)(`ModifyDBClusterParameterGroupAsync`)
+* [Create a DB cluster](Actions/AuroraWrapper.cs#L41) (`CreateDBCluster`)
+* [Create a DB cluster parameter group](Actions/AuroraWrapper.cs#L41) (`CreateDBClusterParameterGroup`)
+* [Create a DB cluster snapshot](Actions/AuroraWrapper.cs#L320) (`CreateDBClusterSnapshot`)
+* [Create a DB instance in a DB cluster](Actions/AuroraWrapper.cs#L286) (`CreateDBInstance`)
+* [Delete a DB cluster](Actions/AuroraWrapper.cs#L181) (`DeleteDBCluster`)
+* [Delete a DB cluster parameter group](Actions/AuroraWrapper.cs#L181) (`DeleteDBClusterParameterGroup`)
+* [Delete a DB instance](Actions/AuroraWrapper.cs#L386) (`DeleteDBInstance`)
+* [Describe DB cluster parameter groups](Actions/AuroraWrapper.cs#L98) (`DescribeDBClusterParameterGroups`)
+* [Describe DB cluster snapshots](Actions/AuroraWrapper.cs#L340) (`DescribeDBClusterSnapshots`)
+* [Describe DB clusters](Actions/AuroraWrapper.cs#L259) (`DescribeDBClusters`)
+* [Describe DB instances](Actions/AuroraWrapper.cs#L236) (`DescribeDBInstances`)
+* [Describe database engine versions](Actions/AuroraWrapper.cs#L21) (`DescribeDBEngineVersions`)
+* [Describe options for DB instances](Actions/AuroraWrapper.cs#L154) (`DescribeOrderableDBInstanceOptions`)
+* [Describe parameters from a DB cluster parameter group](Actions/AuroraWrapper.cs#L66) (`DescribeDBClusterParameters`)
+* [Update parameters in a DB cluster parameter group](Actions/AuroraWrapper.cs#L115) (`ModifyDBClusterParameterGroup`)
 
 ### Scenarios
 
-Code examples that show you how to accomplish a specific task by calling
-multiple functions within the same service.
+Code examples that show you how to accomplish a specific task by calling multiple
+functions within the same service.
+
+* [Get started with DB clusters](Scenarios/AuroraScenario.cs) 
 
-* [Get started with DB clusters](Scenarios/AuroraScenario.cs)
+### Cross-service examples
+
+Sample applications that work across multiple AWS services.
+
+* [Create an Aurora Serverless work item tracker](../cross_service/AuroraItemTracker) 
 
 ## Run the examples
 
 ### Prerequisites
-* To find prerequisites for running these examples, see the
-  [README](../README.md#Prerequisites) in the dotnetv3 folder.
 
-### Get started with DB clusters
+For prerequisites, see the [README](../README.md#Prerequisites) in the `dotnetv3` folder.
+
+
+<!--custom.prerequisites.start-->
+<!--custom.prerequisites.end-->
+
+### Instructions
+
+<!--custom.instructions.start-->
+<!--custom.instructions.end-->
 
-This interactive scenario runs at a command prompt and shows you how to use
-Aurora with the AWS SDK for .NET to do the following:
+#### Hello Aurora
```

Comment:
Should this refer to the Hello Aurora section of the Actions or does it refer to the following scenario description?

Reply:
This is added extra descriptions from the Jinja template generation. It's in the correct spot here as far as I can tell.